### DISCUSSION TITLE
ETA on v6.8.7? Customer-facing improvements anticipated with this release

### DISCUSSION THREAD

**Discussion by atiterlea** [ansible/ansible-lint#2700] [Q&A] [answered]
Do we have an ETA on the next release to PyPI? We are anxious to incorporate some of the customer-facing improvements that will be included in v6.8.7 into our workflows.

**Comment by ssbarnea** [comment, marked as answer]
Released https://github.com/ansible/ansible-lint/discussions/2705 but we will have another one this week as other fixed merged.
